pkg_list.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/python
```

Comment:
What is this module used for?

Reply:
Dependency tooling does not use pkg_list.py. Not quite sure why this script is pulled in this pull request.

Reply:
I have removed "pkg_list.py" from commit.